test/controllers/charityInformation/CanWeSendToThisAddressControllerSpec.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.charityInformation
+
+import base.SpecBase
+import controllers.actions.{AuthIdentifierAction, FakeAuthIdentifierAction}
+import forms.charityInformation.CanWeSendToThisAddressFormProvider
+import models.addressLookup.{AddressModel, CountryModel}
+import models.NormalMode
+import navigation.CharityInformationNavigator
+import navigation.FakeNavigators.FakeCharityInformationNavigator
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, verify, _}
+import org.scalatest.BeforeAndAfterEach
+import pages.addressLookup.CharityInformationAddressLookupPage
+import pages.charityInformation.CanWeSendToThisAddressPage
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.Helpers._
+import repositories.UserAnswerRepository
+import views.html.charityInformation.CanWeSendToThisAddressView
+
+import scala.concurrent.Future
+
+class CanWeSendToThisAddressControllerSpec extends SpecBase with BeforeAndAfterEach {
+
```

Comment:
please check unit scoverage report for this class, if possible add the required tests

Reply:
Added required tests for 100% coverage